### DISCUSSION TITLE
How to defeat the guardian?

### DISCUSSION THREAD

**Discussion by ghost** [LucasHazardous/mAIn_world#9] [Q&A] [answered]
This is a very hard fight. Any tips on how to win will be appreciated.

**Comment by LucasHazardous** [comment, marked as answer]
If you get behind him at the right time, you may be able to avoid a hit. Constantly walk around him, spam katana attack, don't jump and make sure to damage him with EMP. If you manage that, you will eventually defeat him. This is a tough fight so don't expect to win for the first couple of times. Good luck!
